scripts/linters/pylint_extensions.py

```diff
@@ -1163,7 +1164,8 @@ def process_module(self, node):
             if line.count(string_indicator) >= 2:
                 continue
 
-            if re.search(br'[^/]/[^/]', line):
+            if (re.search(br'[^/]/[^/]', line) and
+                    not line.endswith(multi_line_indicator)):
                 self.add_message(
                     'division-operator-used', line=line_num + 1)
 
```

Comment:
Please note that line 1281 is still not covered: 
![image](https://user-images.githubusercontent.com/5094060/83315803-63fd7780-a1f0-11ea-8466-8bcaca430860.png)

Reply:
@brianrodri Strange, this line has [test](https://github.com/oppia/oppia/blob/9d1780352e310b23b4f0148b552ab1bc1343c999/scripts/linters/pylint_extensions_test.py#L1686) for it. Do you have any idea why this error is popping up?

Reply:
Hmm nope I'm not sure, but you can try running that test all by itself and check which lines it *does* hit, might give you a clue:

    python -m scripts.run_backend_tests \
        --test_target=scripts.linters.pylint_extensions_test.SingleLineCommentCheckerTests.test_variable_name_in_comment \
        --generate_coverage_report

Reply:
@brianrodri I think I found the reason behind this unexpected behavior ([see here](https://bitbucket.org/ned/coveragepy/issues/198/continue-marked-as-not-covered)). I am seeing using `# pragma: no cover` as the only option. What are your thoughts on this?

Reply:
Sure, but please provide proof it works and add a detailed comment explaining why the pragma is required.

Reply:
Actually, try not to use "no cover" if possible -- we don't want that to be a pattern that people copy throughout the codebase.

Maybe you can find other ways to rewrite the code such that it still does the same thing, e.g. separating into two if statements?

Reply:
Done.